cbdr/cortex
# COR-741: Make Featured Image Optional

## Purpose:
Changes logic in the media RSS methods to make it so the feed doesn't break if no featured image is present.

> As a content creator, I may choose to write a post that has no featured image and should be allowed to publish through RSS without failure when there is no featured image.

## JIRA:
https://cb-content-enablement.atlassian.net/browse/COR-741

## Steps to Take On Prod
* Create an employer blog with no featured image
* Go to the feed (you may need to clear the cache a few times) and confirm you can see your created Employer Blog *AND* that the feed doesn't error out

## Changes:
* Changes to setup
  * N/A

* Architectural changes
  * N/A

* Migrations
  * N/A

* Library changes
  * N/A

* Side effects
  * N/A

## Screenshots
* Before
N/A

* After
N/A

## QA Links:
http://web.cortex-5.development.c66.me/

## How to Verify These Changes
* Specific pages to visit
  * `/rss/v2/Employer%20Blog`

* Steps to take
  * Confirm it doesn't explode when there's a Employer Blog without a Featured Image

* Responsive considerations
  * N/A

## Relevant PRs/Dependencies:
N/A

## Additional Information
N/A
